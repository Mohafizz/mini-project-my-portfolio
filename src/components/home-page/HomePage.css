.homepage-avatar {
  float: right;
  height: 100%;
  width: auto;
  border-radius: 50%;
  margin-right: 50px;
  padding: 10px;
}

.homepage-text {
  background-color: aliceblue;
  padding: 20px;
  font-family: "Muli", sans-serif;
  text-align: left;
  float: left;
}
