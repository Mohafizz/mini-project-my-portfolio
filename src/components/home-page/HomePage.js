import React from "react";
import "./HomePage.css";
import { Grid, Cell } from "react-mdl";

const HomePage = () => {
  return (
    <div>
      <div style={{ width: "100%", margin: "auto" }}>
        <Grid className="projects-grid">
          <Cell col={4} tablet={6} phone={3}>
            <img
              className="homepage-avatar"
              src="Images/mohafizz_avatar.jpg"
              alt="mohafizz"
            />
          </Cell>

          <Cell col={6} tablet={8}>
            <div className="homepage-text">
              <h3>Hello there!</h3>
              <p>
                My name is Mohamed Hafiz Bin Abu Bakar. This is my journey as a
                Jumpstarter to switch my career into Tech. Please feel free to
                browse to this website to view all the projects that I am still
                embarking.
              </p>
            </div>
          </Cell>
        </Grid>
      </div>
      <br />
    </div>
  );
};

export default HomePage;
