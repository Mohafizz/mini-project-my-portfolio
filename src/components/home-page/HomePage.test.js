import React from "react";
import { shallow } from "enzyme";
import HomePage from "./HomePage";

const homePage = shallow(<HomePage />);

it("should render correctly", () => {
  expect(homePage).toMatchSnapshot();
});
