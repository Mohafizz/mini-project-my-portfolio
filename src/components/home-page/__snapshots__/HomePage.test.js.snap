// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <HomePage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          style={
                    Object {
                              "margin": "auto",
                              "width": "100%",
                            }
          }
>
          <Grid
                    className="projects-grid"
          >
                    <Cell
                              col={4}
                              phone={3}
                              tablet={6}
                    >
                              <img
                                        alt="mohafizz"
                                        className="homepage-avatar"
                                        src="Images/mohafizz_avatar.jpg"
                              />
                    </Cell>
                    <Cell
                              col={6}
                              tablet={8}
                    >
                              <div
                                        className="homepage-text"
                              >
                                        <h3>
                                                  Hello there!
                                        </h3>
                                        <p>
                                                  My name is Mohamed Hafiz Bin Abu Bakar. This is my journey as a Jumpstarter to switch my career into Tech. Please feel free to browse to this website to view all the projects that I am still embarking.
                                        </p>
                              </div>
                    </Cell>
          </Grid>
</div>,
        <br />,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Grid
            className="projects-grid"
>
            <Cell
                        col={4}
                        phone={3}
                        tablet={6}
            >
                        <img
                                    alt="mohafizz"
                                    className="homepage-avatar"
                                    src="Images/mohafizz_avatar.jpg"
                        />
            </Cell>
            <Cell
                        col={6}
                        tablet={8}
            >
                        <div
                                    className="homepage-text"
                        >
                                    <h3>
                                                Hello there!
                                    </h3>
                                    <p>
                                                My name is Mohamed Hafiz Bin Abu Bakar. This is my journey as a Jumpstarter to switch my career into Tech. Please feel free to browse to this website to view all the projects that I am still embarking.
                                    </p>
                        </div>
            </Cell>
</Grid>,
          "style": Object {
            "margin": "auto",
            "width": "100%",
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <Cell
                col={4}
                phone={3}
                tablet={6}
>
                <img
                                alt="mohafizz"
                                className="homepage-avatar"
                                src="Images/mohafizz_avatar.jpg"
                />
</Cell>,
              <Cell
                col={6}
                tablet={8}
>
                <div
                                className="homepage-text"
                >
                                <h3>
                                                Hello there!
                                </h3>
                                <p>
                                                My name is Mohamed Hafiz Bin Abu Bakar. This is my journey as a Jumpstarter to switch my career into Tech. Please feel free to browse to this website to view all the projects that I am still embarking.
                                </p>
                </div>
</Cell>,
            ],
            "className": "projects-grid",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": <img
                  alt="mohafizz"
                  className="homepage-avatar"
                  src="Images/mohafizz_avatar.jpg"
/>,
                "col": 4,
                "phone": 3,
                "tablet": 6,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "mohafizz",
                  "className": "homepage-avatar",
                  "src": "Images/mohafizz_avatar.jpg",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": <div
                  className="homepage-text"
>
                  <h3>
                                    Hello there!
                  </h3>
                  <p>
                                    My name is Mohamed Hafiz Bin Abu Bakar. This is my journey as a Jumpstarter to switch my career into Tech. Please feel free to browse to this website to view all the projects that I am still embarking.
                  </p>
</div>,
                "col": 6,
                "tablet": 8,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <h3>
                      Hello there!
</h3>,
                    <p>
                      My name is Mohamed Hafiz Bin Abu Bakar. This is my journey as a Jumpstarter to switch my career into Tech. Please feel free to browse to this website to view all the projects that I am still embarking.
</p>,
                  ],
                  "className": "homepage-text",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Hello there!",
                    },
                    "ref": null,
                    "rendered": "Hello there!",
                    "type": "h3",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "My name is Mohamed Hafiz Bin Abu Bakar. This is my journey as a Jumpstarter to switch my career into Tech. Please feel free to browse to this website to view all the projects that I am still embarking.",
                    },
                    "ref": null,
                    "rendered": "My name is Mohamed Hafiz Bin Abu Bakar. This is my journey as a Jumpstarter to switch my career into Tech. Please feel free to browse to this website to view all the projects that I am still embarking.",
                    "type": "p",
                  },
                ],
                "type": "div",
              },
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "br",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            style={
                        Object {
                                    "margin": "auto",
                                    "width": "100%",
                                  }
            }
>
            <Grid
                        className="projects-grid"
            >
                        <Cell
                                    col={4}
                                    phone={3}
                                    tablet={6}
                        >
                                    <img
                                                alt="mohafizz"
                                                className="homepage-avatar"
                                                src="Images/mohafizz_avatar.jpg"
                                    />
                        </Cell>
                        <Cell
                                    col={6}
                                    tablet={8}
                        >
                                    <div
                                                className="homepage-text"
                                    >
                                                <h3>
                                                            Hello there!
                                                </h3>
                                                <p>
                                                            My name is Mohamed Hafiz Bin Abu Bakar. This is my journey as a Jumpstarter to switch my career into Tech. Please feel free to browse to this website to view all the projects that I am still embarking.
                                                </p>
                                    </div>
                        </Cell>
            </Grid>
</div>,
          <br />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Grid
              className="projects-grid"
>
              <Cell
                            col={4}
                            phone={3}
                            tablet={6}
              >
                            <img
                                          alt="mohafizz"
                                          className="homepage-avatar"
                                          src="Images/mohafizz_avatar.jpg"
                            />
              </Cell>
              <Cell
                            col={6}
                            tablet={8}
              >
                            <div
                                          className="homepage-text"
                            >
                                          <h3>
                                                        Hello there!
                                          </h3>
                                          <p>
                                                        My name is Mohamed Hafiz Bin Abu Bakar. This is my journey as a Jumpstarter to switch my career into Tech. Please feel free to browse to this website to view all the projects that I am still embarking.
                                          </p>
                            </div>
              </Cell>
</Grid>,
            "style": Object {
              "margin": "auto",
              "width": "100%",
            },
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                <Cell
                  col={4}
                  phone={3}
                  tablet={6}
>
                  <img
                                    alt="mohafizz"
                                    className="homepage-avatar"
                                    src="Images/mohafizz_avatar.jpg"
                  />
</Cell>,
                <Cell
                  col={6}
                  tablet={8}
>
                  <div
                                    className="homepage-text"
                  >
                                    <h3>
                                                      Hello there!
                                    </h3>
                                    <p>
                                                      My name is Mohamed Hafiz Bin Abu Bakar. This is my journey as a Jumpstarter to switch my career into Tech. Please feel free to browse to this website to view all the projects that I am still embarking.
                                    </p>
                  </div>
</Cell>,
              ],
              "className": "projects-grid",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": <img
                    alt="mohafizz"
                    className="homepage-avatar"
                    src="Images/mohafizz_avatar.jpg"
/>,
                  "col": 4,
                  "phone": 3,
                  "tablet": 6,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "mohafizz",
                    "className": "homepage-avatar",
                    "src": "Images/mohafizz_avatar.jpg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": <div
                    className="homepage-text"
>
                    <h3>
                                        Hello there!
                    </h3>
                    <p>
                                        My name is Mohamed Hafiz Bin Abu Bakar. This is my journey as a Jumpstarter to switch my career into Tech. Please feel free to browse to this website to view all the projects that I am still embarking.
                    </p>
</div>,
                  "col": 6,
                  "tablet": 8,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <h3>
                        Hello there!
</h3>,
                      <p>
                        My name is Mohamed Hafiz Bin Abu Bakar. This is my journey as a Jumpstarter to switch my career into Tech. Please feel free to browse to this website to view all the projects that I am still embarking.
</p>,
                    ],
                    "className": "homepage-text",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Hello there!",
                      },
                      "ref": null,
                      "rendered": "Hello there!",
                      "type": "h3",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "My name is Mohamed Hafiz Bin Abu Bakar. This is my journey as a Jumpstarter to switch my career into Tech. Please feel free to browse to this website to view all the projects that I am still embarking.",
                      },
                      "ref": null,
                      "rendered": "My name is Mohamed Hafiz Bin Abu Bakar. This is my journey as a Jumpstarter to switch my career into Tech. Please feel free to browse to this website to view all the projects that I am still embarking.",
                      "type": "p",
                    },
                  ],
                  "type": "div",
                },
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "br",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
